diff --git a/src/nsterm.m b/src/nsterm.m
index 44979c7c04e..eee75641f61 100644
--- a/src/nsterm.m
+++ b/src/nsterm.m
@@ -9021,7 +9021,7 @@ - (instancetype) initWithEmacsFrame: (struct frame *) f
   else if (f->tooltip)
     styleMask = 0;
   else
-    styleMask = (NSWindowStyleMaskTitled
+    styleMask = (NSWindowStyleMaskBorderless
 		 | NSWindowStyleMaskResizable
 		 | NSWindowStyleMaskMiniaturizable
 		 | NSWindowStyleMaskClosable);
@@ -9098,7 +9098,7 @@ - (instancetype) initWithEmacsFrame: (struct frame *) f
         [self setOpaque:NO];
 
       /* toolbar support */
-      [self createToolbar:f];
+      // [self createToolbar:f];
 
       /* macOS Sierra automatically enables tabbed windows.  We can't
          allow this to be enabled until it's available on a Free system.
